core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -2517,7 +2517,7 @@ class GroupCoordinatorTest {
   @Test
   def testDescribeGroupStable() {
     val memberId = JoinGroupRequest.UNKNOWN_MEMBER_ID
-    val joinGroupResult = dynamicJoinGroup(groupId, memberId, protocolType, protocols)
+    val joinGroupResult = staticJoinGroup(groupId, memberId, leaderInstanceId, protocolType, protocols)
```

Comment:
How come we're replacing instead of adding a new test?

Reply:
Because we only need to replace a dynamic member with a static member so that we could retrieve a valid `group.instance.id` for testing purpose. Should be sufficient because all the other test outputs will be expected to be same.

Reply:
Yeah, but we want to test this scenario with both static and dynamic, right?

Reply:
Sounds good to me